test/e2e/framework/test_context.go

```diff
@@ -374,4 +374,8 @@ func AfterReadingAllFlags(t *TestContextType) {
 			t.Host = defaultHost
 		}
 	}
+	// Allow 1% of nodes to be unready on large clusters (statistically).
+	if t.NumNodes >= 100 {
```

Comment:
I thought about it and intentionally left it this way. As otherwise, other people will not be able to set a non-zero value for it even if they want to.

Reply:
So let's do it other way - if set to zero, then overwrite to 1%.

Reply:
Done.